src/gateway/runGateway.ts

```diff
@@ -4,7 +4,8 @@ import {runLoadPeersTask} from "./tasks/loadPeers";
 import {runSyncBlocksTask} from "./tasks/syncBlocks";
 import {runVerifyInteractionsTask} from "./tasks/verifyInteractions";
 
-
+// TODO: I would move the `initGatewayDb` to a seaparate module (e.g. schemas.ts or db.ts)
+// along with the `INTERACTIONS_TABLE` type
 export type INTERACTIONS_TABLE = {
```

Comment:
Probably I'm missing sth, but don't we need to store contractIds for interactWrites to make faster queries?

Reply:
the only thing that is missing here is the internal writes handling ;-), I'm aware of that, planned for the version 0.0.2 ;-)

Reply:
https://github.com/redstone-finance/redstone-sw-gateway/issues/15